### DISCUSSION TITLE
Only 10 basic telemetry sensors on iNav 5/6 with TBS rx or ELRS rx

### DISCUSSION THREAD

**Discussion by EastwoodStudio** [iNavFlight/inav#9062] [Q&A]
When I replaced the previous broken F4V3 with a new F411 wing flight controller, I found that the TBS receiver Telemetry can only receive 10 basic sensor data. After studying issues on github(#6931 #5877 ...) for a while, the problem still cannot be solved.

So I bought more flight controllers and receivers, trying to find out the cause of the problem by comparison.
![WX20230521-102113@2x](https://github.com/iNavFlight/inav/assets/43127991/2740aea2-e1d9-4d78-8664-3f751f2a229f)
Y: with all sensors
N: only 10 base sensors

iNav tried 5.1.0 and 6.1.0
TBS with v6.19 firmware and Hardware is v2.03
ELRS RX tried v3.0.1 and v3.2.1
RC: TX16s with Tbs micro Tx & Betafpv micro 1w ELRS TX

The final result may indicate that there is a difference in hardware or protocol, and I can't locate the specific problem point. The special Tbs problem makes me feel not good, other than Telemetry, other control functions are OK.

There are some common reasons for telemetry that I've tried, such as:
- not turn on TBS telemetry
- not turn on iNav telemetry
- iNav receiver protocol does not choose CRSF (invert OFF / half-duplex auto)
- UART port selection error (my Rx and FC should not be invert)
- Did not delete and refresh telemetry data on the remote side

Hope to get new tips and ideas, thanks

**Comment by OptimusTi** [comment]
This is a common issue related to the setup of TBS or ELSR systems / OPENTX EDGETX and not to INAV.

**Comment by EastwoodStudio** [comment]
I received a new TBS nano pro Rx. It works fine with iNav 6/5 on every FC !

Probably the nano Rx was damaged in some way.  It once flew near a high voltage power line with the FDRC F4V3 FC (which one is broken). The strange thing is that all the remote functions are good except telemetry. What electronic components may be affected by the high field strength.
